actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -77,6 +77,10 @@ module RequestForgeryProtection
       config_accessor :log_warning_on_csrf_failure
       self.log_warning_on_csrf_failure = true
 
+      # Contols whether form-action/method specific CSRF tokens are used.
+      config_accessor :per_form_csrf_tokens_enabled
+      self.per_form_csrf_tokens_enabled = false
```

Comment:
Can this be added as an initializer like you did in #22263 so new Rails apps get this protection by default?